sdk-setup/src/mb2

```diff
@@ -351,7 +351,10 @@ COMMANDS
         effect is cumulative and persists until overriden with later name
         prefixed with the opposite operator.
 
-    deploy {--pkcon|--rsync|--sdk|--zypper|--manual} [--all] [--debug]
+    sign 
+        Generate and insert new signatures for each collected package, replacing any existing signatures.
```

Comment:
The 'sign' command doesn't do anything special in addition to collecting the related rpms. So, instead of introducing a new command, I would prefer adding '--sign' option to the 'build' and 'package' commands.

Reply:
There will be no additional build step on the qtcreator side?

Reply:
'build' and 'package' are the only commands that result in RPM creation (unless I missed something), so only those need the '--sign' option. Hopefully I answered your question as I am not sure I follow.

Reply:
Yes. But what if we want to sign a RPM that has already been created?

Reply:
In that case we simply use rpmsign (or its variant) directly.

Reply:
I update PR: add argument for 'build' and 'package'. By default sign is enable, disable it with '--nosign'. This option is the most correct and convenient, but in this case, in PR-https://github.com/sailfishos/sailfish-qtcreator/pull/325 we will lose the sign build step. I wanted to use it for us, to use a custom signature command with its own parameters and output parsing.